modules/local/checkclinical/main.nf

```diff
@@ -0,0 +1,92 @@
+process CHECK_CLINICAL {
+    tag "${study_id}"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:2.0.1--pyhdfd78af_0':
+        'biocontainers/ampcombi:2.0.1--pyhdfd78af_0' }"
+
+    input:
+        tuple val(study_id)
+        tuple val(token)
+        tuple path(file_metadata) // Spreadsheet
+        tuple path(analysis_metadata) // Spreadsheet
+        tuple path(workflow_metadata) // Spreadsheet
+        tuple path(read_group_metadata) // Spreadsheet
+        tuple path(experiment_metadata) // Spreadsheet
+        tuple path(specimen_metadata) // Spreadsheet
+        tuple path(sample_metadata) // Spreadsheet
+        tuple path(data_directory) // file path
+
+    output:
+        path "*/*/*.{SUCCESS,FAILURE}" , emit: status
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${study_id}"
+    def sample_file = sample_metadata ? "--sample_metadata ${sample_metadata}" : ""
+    def specimen_file = specimen_metadata ? "--specimen_metadata ${specimen_metadata}" : ""
+    def experiment_file = experiment_metadata ? "--experiment_metadata ${experiment_metadata}" : ""
+    def read_group_file = read_group_metadata ? "--read_group_metadata ${read_group_metadata}" : ""
+    def workflow_file = workflow_metadata ? "--workflow_metadata ${workflow_metadata}" : ""
+
+    """
+    main.py \
+        --file_metadata ${file_metadata} \
+        --analysis_metadata ${analysis_metadata} \
+        --clinical_url ${params.clinical_url} \
+        --file_manager_url ${params.file_manager_url} \
+        --token ${token} \
+        --study_id  ${study_id} \
+        --data-directory ${data_directory} \
+        --output-directory ${study_id} \
+        ${sample_file} \
+        ${specimen_file} \
+        ${experiment_file} \
+        ${read_group_file} \
+        ${workflow_file}
+
```

Comment:
I suggest to capture all error output of running the python code into a *.tmp file, and parse the tmp file accordingly to generate the `status.yml`. And the `status.yml` will need to follow the similar convention for the `batch_receipt_generate` subworkflow to parse it. 
Reference implementation: https://github.com/Pan-Canadian-Genome-Library/molecular-data-submission-workflow/blob/integrate_test/modules/local/payload/generate/main.nf#L46-L124